src/Swift.Runtime/src/Metadata/TypeMetadata.cs

```diff
@@ -237,4 +243,10 @@ public override int GetHashCode ()
     {
         return handle.GetHashCode ();
     }
+
+    static TypeMetadataCache cache;
```

Comment:
I believe this could be `readonly` as well.

Reply:
FYI, adding private is redundant. The default visibility for types in C# is private.

Reply:
I know, i think the runtime code guideline recommends explicitly specifying access modifiers

Reply:
Right - the mono style guideline recommends leaving out private entirely.  IIRC, `private` was added to C# for parity with Java not because it was needed.